jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/util/DateUtils.java

```diff
@@ -19,7 +19,6 @@
 import java.util.Date;
```

Comment:
I guess we should also consider moving to ```java.time``` package in the future, as ```java.util.Date``` looks deprecated